pkg/interface/src/views/landscape/components/GroupSettings/GroupFeed.tsx

```diff
@@ -66,7 +66,7 @@ export function GroupFeedSettings(props: {
               </Label>
             </Col>
           </BaseLabel>
-          {isEnabled && false && (
+          {isEnabled && (
```

Comment:
We put this in last night because it doesn’t handle having permissions changed. Does this work now? I’ll try, otherwise this should drop

Reply:
It does work now